### PR TITLE
improved Readme file

### LINKED ISSUE #10770: [Docs] Update Meshery repo README
### Current State
In recent months, Meshery has introduced new features and enhanced its UI. However, the current Meshery repository README still displays outdated screenshots and lacks information about the latest features.

### Desired State
The Meshery repository should showcase the latest features, include up-to-date screenshots and information.

### Implementation
1. Update the Meshery repository README to serve as a comprehensive user guide.
1. Replace outdated screenshots and GIFs with those from the latest version of Meshery.
1. Include additional Meshery features and remove any unnecessary ones from the README.
1. Take inspiration from any good structured README to restructure existing Meshery repo readme

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by praduman8435**
**Notes for Reviewers**

This PR fixes #10770

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,381 +1,484 @@
-
-<p style="text-align:center;" align="center"><a href="https://meshery.io"><picture>
- <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/meshery-logo-light-text-side.svg">
- <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/meshery-logo-dark-text-side.svg">
-<img src="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/meshery-logo-dark-text-side.svg"
-alt="Meshery Logo" width="70%" /></picture></a><br /><br /></p>
-<p align="center">
-<a href="https://hub.docker.com/r/meshery/meshery" alt="Docker pulls">
-  <img src="https://img.shields.io/docker/pulls/layer5/meshery.svg" /></a>
-<a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+org%3A+label%3A%22help+wanted%22+" alt="GitHub issues by-label">
```
Comment (leecalcote) [inline README.md:10]: Thanks, @praduman8435.

I don't know that this relocation is desirable, @Yashsharma1911

Comment (praduman8435) [reply README.md:10]: Hi! @leecalcote
Thank you for your feedback

If you think we should add or take out anything specific, just tell me. I'll check again to see if moving stuff around in the README could be better and make it more desirable. Your input is essential in making sure the README helps everyone

Comment (leecalcote) [reply README.md:10]: @Yashsharma1911 opened the issue. I'll let him offer specifics. I believe that he already did offer much guidance in the open issue.

If this isn't an area of comfort and you need help, @praduman8435, I don't know that there's any to offer you that wouldn't make sense for another contributor to just execute themselves. In other words, of the time that I were to take explaining it, I would do better to just execute the request. 

Should we reassign the issue or can you handle it?

Comment (praduman8435) [reply README.md:10]: Apologies if there were any misunderstanding, I'm comfortable handling this myself. Soon I will update the PR

**Comment by praduman8435** [comment]
@Yashsharma1911 , I have updated the PR

**Review by ShivamCoder23 — CHANGES_REQUESTED** [review]
@praduman8435 conflicts

**Comment by praduman8435** [comment]
> @praduman8435 conflicts

resolved conflicts

File: .github/assets/videos/Intro-to-meshery.webm
Comment (leecalcote) [inline .github/assets/videos/Intro-to-meshery.webm:1]: Please remove this video from being directly embedded.

**Comment by leecalcote** [comment]
@Yashsharma1911, let's find @praduman8435 another issue to work on. The issue that this PR links to is not a "good first issue" for the uninitiated to the Meshery project. We're just spinning our wheels here. A maintainer will need to pick this up.

**Pull request closed without merging**